spec/yaml/swiftnav/sbp/system.yaml

```diff
@@ -66,3 +66,6 @@ definitions:
                   values:
                       - 0: System Healthy
                       - 1: An error has occurred
+        - protocol:
```

Comment:
So we'd have 24 bits in the reserved field of the heartbeat message? And use a similar fields scheme for the new bootloader handshake? Makes sense to be more formal and rigid with the SBP protocol version. Any objections?